admin/test/config/browserstack.yml

```diff
@@ -0,0 +1,75 @@
+server: "hub-cloud.browserstack.com"
+user: mohsen20
+authkey: ZyZxbzNCWSdnp1sgLCEz
```

Comment:
@activemq We certainly can't check this in - you have just provided your authkey to the world - You should revoke this key asap!